timefold-solver/README.md

```diff
@@ -0,0 +1,7 @@
+## OptaPlanner
+
+This module contains articles about OptaPlanner.
+
+### Relevant articles
+
+- [A Guide to OptaPlanner](https://www.baeldung.com/opta-planner)
```

Comment:
We can omit this line for now. The editorial team will add it after publishing once they have the final URL.

Reply:
Yes - and the readme needs to talk about Timefold instead of OptaPlanner (copy-paste error). Fixed.